docs/gettingstarted.asciidoc

```diff
@@ -356,10 +366,24 @@ as argument of the load.sh command line:
 ./load.sh http://192.168.33.60:9200
 -------------------------------------------------------------------------
 
+The load command uploads the example dashboards and the visualization, searches that are used.
```

Comment:
I didn't fully get this sentence. Here a suggestion:
```
The load command uploads the example dashboards with the visualizations and searches that can be used.
```

@monicasarbu I assume we stick with US english?